test/test_install_and_use.sh

```diff
@@ -127,19 +135,23 @@ for ((test_iter=0; test_iter<${tests_count}; ++test_iter )) ; do
     || error_and_proceed "## Param Test ${test_num}/${tests_count}: ${desc} ( ${k} / ${v} ) failed";
 done;
 
-for ((test_iter=0; test_iter<${tests_count}; ++test_iter )) ; do
+tests_file__desc=("${tests__desc[@]}" "${tests_file_only__desc[@]}");
+tests_file__kv+=("${tests__kv[@]}" "${tests_file_only__kv[@]}");
```

Comment:
I think it works either way, but is there any reason for L138 to be assignment, and L139 to be addition?

Reply:
I don't think that can have been deliberate, fixed.